core/src/main/scala/precog/AutoBump.scala

```diff
@@ -300,6 +306,13 @@ class AutoBump(
           NewPullRequestData(autoBumpCommitTitle(authorRepository), description),
           updateBranch.name,
           "master")
+        .handleErrorWith(_ =>
```

Comment:
This assumes that an error is thrown if the base branch does not exist. 

https://docs.github.com/en/rest/reference/pulls suggests that it will

> **Required**. The name of the branch you want the changes pulled into. This should be an existing branch on the current repository.

but still need to verify that it actually does that too.

Reply:
It seems it doesn't throw an exception and [instead we need to check this `Either`](https://github.com/47degrees/github4s/blob/main/github4s/src/main/scala/github4s/GHResponse.scala#L20) to handle the error.

Reply:
Oh right, thanks again! I wrongly thought I could deduplicate the `.rethrowGHError("draftPullRequest")`